resources/views/components/loadout-stats.blade.php

```diff
@@ -0,0 +1,55 @@
+
+@php
+
+    $currentPickRate = $calculatePercentage($entity->loadouts_count, $totalLoadouts);
+    $previousPickRate = $calculatePercentage($comparison->loadouts_count, $previousTotal);
+
+    $pickRateDifference = $calculateDifference($currentPickRate, $previousPickRate);
+
+@endphp
+
+<div class="flex items-center md:space-x-4">
+    <div class="w-1/4">
+        {{ $identifier }}
+    </div>
+    <dl class="mt-5 rounded-lg bg-gray-700 text-gray-200 overflow-hidden shadow divide-y divide-gray-900 w-3/4">
+        <a href="{{ $loadoutLink }}">
+            <div class="px-4 py-5 sm:p-6">
+                <dt class="text-base font-normal text-gray-100">Total Loadouts</dt>
+                <dd class="mt-1 flex justify-between items-baseline md:block lg:flex">
+                    <div class="flex items-baseline text-2xl font-semibold text-orange-600">
+                        {{ $entity->loadouts_count }}
+                    </div>
+                </dd>
+            </div>
+        </a>
+
+
+        <div class="px-4 py-5 sm:p-6">
+            <dt class="text-base font-normal text-gray-100">Pick Rate</dt>
+            <dd class="mt-1 flex justify-between items-baseline md:block lg:flex">
+                <div class="flex items-baseline text-2xl font-semibold text-orange-600">
+                    {{ $currentPickRate }}%
+                    <span class="ml-2 text-sm font-medium text-gray-400"> from {{ $previousPickRate }}% </span>
```

Comment:
Sent a suggestion over slack for a possible improvement here:
![image](https://user-images.githubusercontent.com/1916366/175406086-6bdeca76-a25a-42ae-862d-b8c204e672a6.png)

The "from" wasn't intuitive my first time viewing. I had to actually look into our code to realize we're comparing to previous patch.

We could even say "previous patch".

I think a small UI change like that will improve the ability to immediately understand the statistic. Not required, though